Aspire BudgetingTests/UserManagerTests.swift

```diff
@@ -13,12 +13,12 @@ import XCTest
 
 @testable import Aspire_Budgeting
 
-class UserManagerTests: XCTestCase {
+final class UserManagerTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/20b2e363fa9f372765b4b610131dc68d3ab847c2/Aspire BudgetingTests/UserManagerTests.swift#L16"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>